### DISCUSSION TITLE
New Email Forwarder Integration - Forward Email (PR already tested!  Working screenshots!)

### DISCUSSION THREAD

**Discussion by titanism** [bitwarden/clients#4924] [Password Manager]
### Code Contribution Proposal

A member of your team reached out to us and inquired about our community and service &ndash; since other email forwarding services are already added to Bitwarden (e.g. AnonAddy, SimpleLogin, etc).  We're a 100% open-source email forwarding service at https://forwardemail.net with our source entirely available at https://github.com/forwardemail.

To save everyone time, our team submitted this PR at <https://github.com/bitwarden/clients/pull/4809>, and then ran the local Bitwarden server and tested all the integrations.

Below are screenshots of this PR in action:

> Browser extension modal showing config:
<img width="388" alt="Modal-showing-config" src="https://user-images.githubusercontent.com/101466223/222826195-99368537-759e-46f4-9570-37ff8b5c4f7b.png">

> Forward Email website interface showing addresses generated with "Generated by Bitwarden." description.

<img width="1160" alt="Forward-email-emails-addresses-generated" src="https://user-images.githubusercontent.com/101466223/222826258-193643df-8b46-43ae-9ff2-ea4515e89308.png">

> Browser extension generator showing successfully generated address (with default "hideaddress.net" and API token from the user's security page at https://forwardemail.net/my-account/security):
<img width="436" alt="modal-showing-config-generated-email" src="https://user-images.githubusercontent.com/101466223/222826440-0abd2b36-b320-408f-bffd-fd87e28c1eb8.png">

> Web vault showing config screen for generator with Forward Email in dropdown for "Service" input type.
<img width="1106" alt="web-showing-config-screen" src="https://user-images.githubusercontent.com/101466223/222826546-bbf96cd5-0dd1-43ad-8d4e-fd316f92ef6c.png">

> Web vault showing successfully generated email address using Forward Email and API token:
<img width="1062" alt="web-showing-email-address-generated" src="https://user-images.githubusercontent.com/101466223/222826596-8449f2a7-0cff-46c5-908e-beb74388eb8a.png">

Can you please merge and release this?  I'm sure the Bitwarden community would love to be able to use Forward Email &ndash; and vice versa &ndash; we'd definitely share with our users that they can leverage Bitwarden for password management _and_ also use Forward Email with the same app, across mobile, desktop, and browser.

**Comment by dwbit** [comment]
Thanks @titanism, we will review and get back to you shortly!

**Comment by dwbit** [comment]
@titanism thanks for your patience, the team has confirmed they would like to move forward, stay tuned for next steps! 🚀

**Reply by titanism** [reply]
@dwbit thanks!  If you do merge/release, then you could update this link here too <https://bitwarden.com/blog/add-privacy-and-security-using-email-aliases-with-bitwarden/> and add our name ("Forward Email"), logo, and [link](https://forwardemail.net) (e.g. `As of March 2023 release...`).  Here's our media asset <https://raw.githubusercontent.com/forwardemail/forwardemail.net/master/media/header.png>.

```diff
+Update: As of the [September 2022](https://bitwarden.com/help/releasenotes/), [October 2022](https://bitwarden.com/help/releasenotes/), and [March 2023](https://bitwarden.com/help/releasenotes/) releases, the Bitwarden forwarded email alias generator also integrates with [Fastmail](https://www.fastmail.com/), [DuckDuckGo](https://duckduckgo.com/about), and [Forward Email](https://forwardemail.net). Learn more about this integration in our [help center](https://bitwarden.com/help/generator/#username-types).
-Update: As of the [September 2022](https://bitwarden.com/help/releasenotes/) and [October 2022](https://bitwarden.com/help/releasenotes/) releases, the Bitwarden forwarded email alias generator also integrates with [Fastmail](https://www.fastmail.com/) and [DuckDuckGo](https://duckduckgo.com/about). Learn more about this integration in our [help center](https://bitwarden.com/help/generator/#username-types).
```

**Comment by dwbit** [comment]
Thanks @titanism I'll be sure to loop the team in regarding those changes!

**Comment by djsmith85** [comment]
Closing this discussion as the support was added with `2023.6`
Thanks again @titanism for your contributions ❤️

**Reply by titanism** [reply]
Thank you @djsmith85 !!! Also some exciting news; we now support IMAP, POP3, SMTP, and more.  We're adding calendar and contacts soon, as well as newsletter support.  Happy to help out Bitwarden or anyone else if needed!  See https://forwardemail.net.  Happy new year @djsmith85 and thank you so much for your efforts here 🎉 🙏 🙇

**Reply by withinfocus** [reply]
@titanism if that implies an account upgrade for us Bitwarden engineers, consider me interested 😜

**Reply by titanism** [reply]
We aren't charging extra for these features - they're included with any existing premium FE account.  If any Bitwarden Engineer wants 3+ years free, please email `support@forwardemail.net` from the account you sign up with at https://forwardemail.net (account signup is free, and we support Apple login, Google login, Github login, email login, and passkeys).
